PocketKit/Sources/PocketKit/MyList/ArchivedItemsList/ArchivedItemsListViewModel.swift

```diff
@@ -342,23 +345,23 @@ extension ArchivedItemsListViewModel {
 extension ArchivedItemsListViewModel {
     func shouldSelectCell(with cell: ItemsListCell<ItemIdentifier>) -> Bool {
         switch cell {
-        case .filterButton:
-            return true
-        case .item(let objectID):
-            return !(savedItem(objectID)?.isPending ?? true)
-        case .emptyState, .offline, .placeholder, .tag:
-            return false
+            case .filterButton:
+                return true
+            case .item(let objectID):
+                return !(savedItem(objectID)?.isPending ?? true)
+            case .emptyState, .offline, .placeholder, .tag:
+                return false
```

Comment:
might want to revert all the indentation changes since we don't need to modify the structure and we don't need to touch the code

Reply:
Would love to see us use swiftlint to prevent this.

Reply:
yes! suggested it for the next rounds and roux agrees :) 
https://mozilla.slack.com/archives/CPF01HMUZ/p1666792528020509